src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/ClientsApiServiceImpl.scala

```diff
@@ -123,4 +129,40 @@ final case class ClientsApiServiceImpl(authorizationProcessService: Authorizatio
       }
     }
   }
+
+  override def getClientKeys(clientId: String)(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem],
+    toEntityMarshallerPublicKeys: ToEntityMarshaller[PublicKeys]
+  ): Route = {
+
+    def getRelationship(key: AuthorizationProcessModel.ReadClientKey): Future[PublicKey] =
```

Comment:
```suggestion
    def decorate(key: AuthorizationProcessModel.ReadClientKey): Future[PublicKey] =
```

Reply:
Very cool!

Reply:
You're right, the commit was blocked because other stories had to be pushed to DEV first